Framework/PythonInterface/mantid/api/src/Exports/Axis.cpp

```diff
@@ -24,7 +24,7 @@ using Mantid::Kernel::Unit_sptr;
 using Mantid::specnum_t;
 using namespace boost::python;
 
-GET_POINTER_SPECIALIZATION(Axis)
+// GET_POINTER_SPECIALIZATION(Axis)
```

Comment:
I think the removal of these is the cause of the MSVC build errors. As `GET_POINTER_SPECIALIZATION` is our own macro can we just make this a no op for 2017?

Reply:
I guess so but it was not disabled in all cases. But I will see what happens when I do what you suggest.